cmd/vela-worker/flags.go

```diff
@@ -26,6 +26,13 @@ func flags() []cli.Flag {
 			Usage:   "set hostname for the worker",
 		},
 
+		&cli.Int64Flag{
```

Comment:
Why not utilize the [cli.DurationFlag](https://github.com/urfave/cli/blob/master/flag_duration.go#L10)?

Reply:
This is just being used in time.Sleep so either should work fine. I wasn't sure that more granular control was useful so I never updated this after my initial implementation. I think it would be fine to use either Int64 or DurationFlag if there is any preference.

Reply:
I think duration would be more ideal since it's more user friendly to the administrator.